docs/Install/HPE_Swarm_Learning_installation.md

```diff
@@ -2,6 +2,6 @@
 
 1.   [Download the License Server](Download_the_License_Server.md) 
 2.   [Install the License Server and download Swarm Learning](Install_the_License_Server.md) 
-3.   [Install HPE Swarm Learning](Install_HPE_Swarm_Learning.md) 
+3.   [Installing HPE Swarm Learning Management UI](Installing_HPE_Swarm_Learning_Management_UI.md) 
```

Comment:
This is incorrect. ( this content is already part of point 2) Please remove 3.

We need to add a new 
3. Running SLM-UI Installer  and point to pages 11 to 14 install guide.